applications/nrf_desktop/configuration/nrf54lm20dk_nrf54lm20a_cpuapp/prj.conf

```diff
@@ -93,6 +98,18 @@ CONFIG_LED_PWM=y
 CONFIG_STREAM_FLASH=y
 CONFIG_IMG_MANAGER=y
 CONFIG_MCUBOOT_IMG_MANAGER=y
+# Enable retained memory and retention to enable the Image Manager in the MCUboot RAM load mode.
+# The RAM region with the bootloader info is not retained across reboots as it is not needed.
+# The MCUboot bootloader always provisions this RAM region before the application boots.
+# The RAM retention is disabled at the SoC level (CONFIG_RETAINED_MEM_NRF_RAM_CTRL=n) to optimize
```

Comment:
`The RAM retention that is enabled at the SoC level is disabled in application configuration`?

Reply:
`to optimize power consumption by allowing to disable unused RAM blocks in the power off state`?